README.md

```diff
@@ -1,31 +1,35 @@
-Talent Network
+# Talent Network
```

Comment:
Use Github markdown (hashes, in this case) to create headers and otherwise format your README properly.